core/market/src/negotiation/requestor.rs

```diff
@@ -482,7 +487,7 @@ pub async fn proposal_receiver_thread(
                 .save_initial_proposal(proposal)
                 .await?;
 
-            log::debug!(
+            log::info!(
```

Comment:
It is important to connect Offer/Demand subscriptions with generated Proposal id. We can move it back to debug, if it will spam to much (Or I can even revert it now if you insist). But I completely disagree with idea, that dwarf messages like in line 481 (prev 476) deserve to be on info level, because they are short and meaningful logs should be on debug.

Reply:
ok, so lets log both on debug

Reply:
We decided to leave it on info level.